purchase_cancel_reason/readme/CONFIGURE.md

```diff
@@ -0,0 +1,17 @@
+1. Go to Menu Purchase -> Configuration -> Purchase Order Cancel Reason
+
+![Purchase  -> configuration -> Purchase Order Cancel Reason](../static/description/purchase_cancel_reason_config_1.png)
+
+2. By default you have three Purchase Cancel Reason available:
+ - Just for Quotation
+ - Service no longer needed
+ - Other Service Provider selected
+
+3. Create a new Purchase Cancel Reason
+Click on 'New'
+
+![Alt text](../static/description/purchase_cancel_reason_config_2.png)
```

Comment:
Would you consider adding an alt text for accessibility purposes? Like `Screenshot of the cancel reason list view` in this case.